Diagnostic/diagnostic.py

```diff
@@ -728,8 +728,6 @@ def report_new_mdsd_errors(err_file_path, last_error_time):
     if len(last_error) > 0 and (datetime.datetime.now() - last_error_time) < datetime.timedelta(minutes=30):
         # Only recent error logs (within 30 minutes) are reported.
         hutil.log("Error in MDSD:" + last_error)
-        hutil.do_status_report(g_ext_op_type, "success", '1',
```

Comment:
We'd need someone with knowledge of the original intent to comment on reporting errors with "success" demarkation and what scenarios we are going to lose by fully cutting it out if any at all.

If we learn we are OK to cut it out without degrading any of the supported scenarios, we should be goog to go. If not - we can complicate this change a little by only skipping known false-positives like "table already exists".